uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeavePeriodEntitlementTest.php

```diff
@@ -382,8 +382,8 @@ public function testCanSaveALeavePeriodEntitlementFromAnEntitlementCalculation()
     // 10 + 1 + 3 (Pro Rata + Brought Forward + No. Public Holidays)
     $this->assertEquals(14, $periodEntitlement->getEntitlement());
 
-    //The 3 days deducted because of the Public Holidays
-    $this->assertEquals(-3, $periodEntitlement->getLeaveRequestBalance());
+    //No leave request should be created
+    $this->assertEquals(0, $periodEntitlement->getLeaveRequestBalance());
```

Comment:
You can delete this assertion completely. No need to assert something that the method doesn't do anymore (and it was never supposed to do)